server/src/main/java/org/jboss/as/server/logging/ServerLogger.java

```diff
@@ -1060,8 +1060,8 @@ OperationFailedException illegalCombinationOfHttpManagementInterfaceConfiguratio
     IllegalStateException serverAlreadyPaused();
 
     @LogMessage(level = INFO)
-    @Message(id = 211, value = "Suspending server")
-    void suspendingServer();
+    @Message(id = 211, value = "Suspending server with %dms timeout.")
```

Comment:
Hi Brian, I totally agree! I was just copying what is being done in the rest of the wildfly-core project (with 2 exceptions), namely:

```
[rhusar@syrah wildfly-core]$ git grep %dms
server/src/main/java/org/jboss/as/server/logging/ServerLogger.java:    @Message(id = 25, value = "%s started in %dms - Started %d of %d services (%d services are lazy, passive or on-demand)")
server/src/main/java/org/jboss/as/server/logging/ServerLogger.java:    @Message(id = 26, value = "%s started (with errors) in %dms - Started %d of %d services (%d services failed or missing dependencies, %d services are lazy, passive or on-demand)")
server/src/main/java/org/jboss/as/server/logging/ServerLogger.java:    @Message(id = 28, value = "Stopped deployment %s (runtime-name: %s) in %dms")
server/src/main/java/org/jboss/as/server/logging/ServerLogger.java:    @Message(id = 50, value = "%s stopped in %dms")
server/src/main/java/org/jboss/as/server/logging/ServerLogger.java:    @Message(id = 208, value = "Stopped subdeployment (runtime-name: %s) in %dms")
server/src/main/java/org/jboss/as/server/logging/ServerLogger.java:    @Message(id = 211, value = "Suspending server with %dms timeout.")
```

Do you want me to fix this for the whole project then? I am all up for that..